photutils/02_photutils_source_detection.ipynb

```diff
@@ -0,0 +1,1034 @@
+    "dao_findpeaks_match = rounded_centroids_dao.intersection(rounded_centroids_findpeaks)\n",
+    "iraf_findpeaks_match = rounded_centroids_iraf.intersection(rounded_centroids_findpeaks)\n",
+    "\n",
+    "print('''Matching sources found by:\n",
+    "    All methods: {}\n",
+    "    DAOStarFinder & IRAFStarFinder: {}\n",
+    "    DAOStarFinder & find_peaks: {}\n",
+    "    IRAFStarFinder & find_peaks: {}'''\n",
+    "      .format(len(all_match), len(dao_iraf_match), \n",
+    "              len(dao_findpeaks_match), len(iraf_findpeaks_match)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And just for fun, let's plot these matching sources. (The colors chosen to represent different sets are from [Paul Tol's guide for accessible color schemes](https://personal.sron.nl/~pault/).)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up the figure with subplots\n",
+    "fig, ax1 = plt.subplots(1, 1, figsize=(8, 8))\n",
+    "\n",
+    "# Plot the data\n",
+    "fitsplot = ax1.imshow(np.ma.masked_where(xdf_image.mask, xdf_image_clipped), norm=norm_image)\n",
+    "ax1.scatter([x for x, y in list(dao_findpeaks_match)], [y for x, y in list(dao_findpeaks_match)],\n",
+    "            s=30, marker='s', lw=1.5,  facecolor='None', edgecolor='#EE7733',\n",
+    "            label='Found by DAO \\& find\\_peaks')\n",
+    "ax1.scatter([x for x, y in list(dao_iraf_match)], [y for x, y in list(dao_iraf_match)],\n",
+    "            s=30, marker='D', lw=1.5, facecolor='None', edgecolor='#EE3377',\n",
+    "            label='Found by DAO \\& IRAF')\n",
+    "ax1.scatter([x for x, y in list(iraf_findpeaks_match)], [y for x, y in list(iraf_findpeaks_match)],\n",
+    "            s=30, marker='o', lw=1.5, facecolor='None', edgecolor='#0077BB',\n",
+    "            label='Found by IRAF \\& find\\_peaks')\n",
+    "ax1.scatter([x for x, y in list(all_match)], [y for x, y in list(all_match)],\n",
+    "            s=30, marker='o', lw=1.2, linestyle=':',facecolor='None', edgecolor='#BBBBBB',\n",
+    "            label='Found by all methods')\n",
+    "ax1.legend()\n",
+    "\n",
+    "# Define the colorbar\n",
+    "cbar = plt.colorbar(fitsplot, fraction=0.046, pad=0.04, ticks=LogLocator(subs=range(10)))\n",
+    "labels = ['$10^{-4}$'] + [''] * 8 + ['$10^{-3}$'] + [''] * 8 + ['$10^{-2}$']\n",
+    "cbar.ax.set_yticklabels(labels)\n",
+    "\n",
+    "# Define labels\n",
+    "cbar.set_label(r'Flux Count Rate ($e^{-1}/s$)', rotation=270, labelpad=30)\n",
+    "ax1.set_xlabel('X (pixels)')\n",
+    "ax1.set_ylabel('Y (pixels)')\n",
+    "ax1.set_title('Sources Found by Different Methods')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Remember that you can double-click on the plot to zoom in and look around!*"
```

Comment:
I'm confused by this.  Is that actually true in general for most browsers? (I haven't had a chance to check this in the live notebook, but will check later to be sure)